app/Http/Controllers/MaterialController.php

```diff
@@ -2,69 +2,82 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\Employee;
 use App\Models\Material;
 use Illuminate\Http\Request;
-use Illuminate\Support\Facades\Route;
 
 class MaterialController extends Controller
 {
-    public function index()
+    const VALIDATION_RULES = [
+        'subject' => 'required',
+        'category' => 'required',
+        'text' => 'required',
```

Comment:
Нужно ещё добавить валидацию для обязательного поля "mentor_uuid". 
```suggestion
        'text' => 'required',
        'mentor_uuid' => 'required|exists:employees,uuid',
```